textdb/textdb-dataflow/pom.xml

```diff
@@ -59,13 +59,16 @@
             <version>3.6.0</version>
             <classifier>models</classifier>
         </dependency>
-
-
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-all</artifactId>
             <version>1.9.5</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
```

Comment:
I agree with @zuozhiw that we want to minimize the number of depending packages.  Besides being faster, does `gson` have other benefits?  Is it worth the addition?